src/handlers/assign.rs

```diff
@@ -56,7 +57,11 @@ const WELCOME_WITHOUT_REVIEWER: &str = "@Mark-Simulacrum (NB. this repo may be m
 
 const RETURNING_USER_WELCOME_MESSAGE: &str = "r? @{assignee}
 
-({bot} has picked a reviewer for you, use r? to override)";
+{bot} has assigned @{assignee}.
+They will have a look at your PR within the next two weeks and either review your PR or
+reassign to another reviewer.
```

Comment:
Need a line break, or it will look not nice, https://github.com/rust-lang/rust/pull/120750#issuecomment-1932731369

Reply:
Whoops. Will open a PR tomorrow